meinberlin/react/livequestions/QuestionForm.jsx

```diff
@@ -65,36 +65,45 @@ export default class QuestionForm extends React.Component {
 
   render () {
     return (
-      <form id="id-comment-form" action="" onSubmit={this.addQuestion.bind(this)}>
+      <>
         <h2>{askQuestionStr}</h2>
-        {Object.keys(this.props.category_dict).length > 0
-          ? <CategorySelect
-              onSelect={this.selectCategory.bind(this)}
-              category_dict={this.props.category_dict}
+        <form id="id-comment-form" className="form--base panel--heavy" action="" onSubmit={this.addQuestion.bind(this)}>
+          <div className="form-group">
+            {Object.keys(this.props.category_dict).length > 0
+              ? <CategorySelect
+                  onSelect={this.selectCategory.bind(this)}
+                  category_dict={this.props.category_dict}
+                />
+              : ''}
+          </div>
+          <div className="form-group">
+            <label htmlFor="questionTextField">{questionStr}*</label>
+            <textarea
+              placeholder={yourQuestionStr}
+              id="questionTextField"
+              className="form-control"
+              name="questionTextFieldName"
+              rows="3"
+              onChange={this.handleTextChange.bind(this)}
+              required="required"
+              value={this.state.question}
+              maxLength="1000"
```

Comment:
it might be a good idea to label the textarea with the character counter. here is a good example https://design-system.service.gov.uk/components/character-count/

i believe they enhance it with js, for the dynamic value to be read out loud only once you stop typing (probably overkill for this example)